command.go

```diff
@@ -2717,7 +2717,7 @@ func (cmd *ClusterSlotsCmd) readReply(rd *proto.Reader) error {
 
 				nodes[j].Addr = net.JoinHostPort(ip, port)
 
-				if n == 3 {
+				if n >= 3 {
```

Comment:
If we have 4 replies, then we should also read (and probably discard) the 4th reply as well?